docs/src/basic-tutorials.md

```diff
@@ -31,6 +31,15 @@ Note that before you put anything in the scene, it will be blank!
 
 The user-facing functions of Makie are pretty well documented, so you can usually use the help mode in the REPL, or your editor of choice. If you countinue to have issues, see [Getting Help](@ref).
 
+## Plotting in Scripts
```

Comment:
I like this section a lot. It might help people making the transition from plots to makie!